go.mod

```diff
@@ -39,11 +39,11 @@ require (
 	github.com/magefile/mage v1.15.0
 	github.com/mitchellh/hashstructure v1.1.0
 	github.com/oklog/ulid/v2 v2.1.0
-	github.com/open-telemetry/opentelemetry-collector-contrib/extension/healthcheckextension v0.109.0
-	github.com/open-telemetry/opentelemetry-collector-contrib/extension/pprofextension v0.109.0
-	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/jaegerreceiver v0.109.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/extension/healthcheckextension v0.110.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/extension/pprofextension v0.110.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/jaegerreceiver v0.110.0
 	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/prometheusreceiver v0.109.0
```

Comment:
Some of the dependencies are bumped, but some are not. I would expect all the dependencies to be bumped. :thinking:

Reply:
Hmmm, yes, that's odd.  Let me try to ask dependabot to recreate this PR and see if that helps.

Reply:
The 'dependabot recreate' command doesn't seem to have helped.

For now, I updated the remaining dependencies with a simple `sed -i 's/ v0.109.0$/ v0.110.0/' go.mod` command. I propose to wait until next release of OTelCol to see what Dependabot does and if the problem persists, we'll take action then.

Reply:
Note that it's probably not worth putting much effort in tweaking Dependabot configuration, as we will probably be switching to Renovate.